apps/site/assets/ts/__v3api.d.ts

```diff
@@ -352,10 +351,12 @@ export interface RapidTransitHours {
   week: [StopHours[], StopHours[]];
   saturday: [StopHours[], StopHours[]];
   sunday: [StopHours[], StopHours[]];
+  special_service: { [key: string]: [StopHours[], StopHours[]] }[];
```

Comment:
Right now the data type is like this:
```
[
  {"2022-12-24": [StopHours[], StopHours[]]},
  {"2022-12-25": [StopHours[], StopHours[]]},
  ...
]
```

I am thinking of changing it to this:
```
{
  "2022-12-24":  [StopHours[], StopHours[]],
  "2022-12-25": [StopHours[], StopHours[]],
  ...
}
```

This should make searching for the correct dates hours easier on the front end, though I think the backend might become a little more complex.  I am looking into it, I just wanted peoples thoughts

Reply:
I think this would be an improvement.